apps/client/sttr_build/config.ts

```diff
@@ -0,0 +1,21 @@
+import { APIConfig } from "./types.d.ts";
+
+export const apis: APIConfig[] = [
+  {
+    version: 1,
+    outputDir: "sttr",
+    host: "https://sttr-builder.eu.meteorapp.com/api",
+    topics: {
+      "dakkapel-plaatsen": ["WKPxKx4YBJ5fqYSni", "Aa2EX3YprpZQ65non"],
+      "dakraam-plaatsen": ["hMwHKR7Wz4FP8Dm4x", "dRy4PfDs7jQPc9gMG"],
+      "kozijnen-plaatsen-of-vervangen": [
+        "Xm2WwYeGkNN9w6rgQ",
+        "5CJrbgbWZP6uZsouY",
+      ],
+      "zonnepanelen-of-zonneboiler-plaatsen": [
+        "TRhZrFexMjBW42Ky6",
+        "skPG9qqTqWyX9tSY7",
+      ],
```

Comment:
Very very very nice improvement!

Maybe you can already add the new topic, or we can do this in a new PR so we can also add to Contribution how to add a new topic, for possible future devs

Reply:
Will add zonwering.
`Contributing.md` improvements can be separate PR.